rollup/circuitcapacitychecker/impl_test.go

```diff
@@ -0,0 +1,47 @@
+package circuitcapacitychecker
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/scroll-tech/go-ethereum/core/types"
+)
+
+type traceContainer struct {
+	Jsonrpc string
+	Id      int
+	Result  *types.BlockTrace
+}
+
+func BenchmarkApplyBlock(b *testing.B) {
+	ccc := NewCircuitCapacityChecker(false)
```

Comment:
```suggestion
	ccc := NewCircuitCapacityChecker(true)
```

This is the realistic call. I guess it will be even cheaper this way, let's check.

Reply:
ok

Reply:
updated, no significant change in the timings.